### DISCUSSION TITLE
Problem Mydumper

### DISCUSSION THREAD

**Discussion by sebleje76** [mydumper/mydumper#1444] [General]
Hello,

I have a problem executing a backup of a schema containing about 850 tables with mydumper. When I run mydumper command I get an error like "ERROR] - Error dumping schemas (middle_office.xxxx.xxx): Lost connection to MySQL server during query" on my entire schema.
Here are the parameters to pass with the mydymper command.

$MYDUMPER -h yy.Yy.yy.yy -u xxx --password=xxx -P3306 -t 1 -r 10000 --database middle_xx -v 4 -G -R -W -l 500 --no-locks --outputdir /data_bck

Knowing that the database is responding while the mydumper script is running and that the script is focusing on other schemas. I can't find the problem.
Can you help me with this?

**Comment by davidducos** [comment]
Hi @sebleje76, maybe an external process could be killing mydumper connections?

**Reply by sebleje76** [reply]
Hi @davidducos 

Can this be solved with the Mysql options net_read_timeout and net_write_timeout because the mydumper command is executed on a remote server?

**Reply by davidducos** [reply]
Can you share more information about your environment? as we saw this issue in the past and I was not able to develop a reproducible test case.

**Reply by sebleje76** [reply]
the architecture consists of a Mariadb 10.3 server hosted on a docker platform. The schema to be backed up contains around 840 tables with a volume of around 300GB.
The server on which we're running the backup is an Oracle Linux 9.2 server with a version of Mydumper 0.15.1.
If you need more info, I'll send it to you.

**Reply by davidducos** [reply]
Can you test with the latest release?

**Reply by sebleje76** [reply]
I will try to update the Mydumper version tomorrow. Because I have backup jobs that will run during the night.

**Comment by sebleje76** [comment]
@davidducos 
I have just upgraded the version of Mydumper, I will test the backup of my schematic which is causing me problems on Sunday.
